backend/src/psihofon/models/answer.py

```diff
@@ -27,6 +26,10 @@ class Meta:
         related_name='answers'
     )
 
+    def __str__(self):
+        return str(_(f"Odgovor {self.order_number}"))
```

Comment:
I suggest changing this to `return f"str(_('Odgovor')) {self.order_number}"` because there can be problems with translations when using string formatting in the way that you implemented it.

Reply:
Got it, thanks.